source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -247,22 +241,7 @@ void BioXASAppController::setupExporterOptions()
 		AMAppControllerSupport::registerClass<BioXASXASScanConfiguration, AMExporterXDIFormat, AMExporterOptionXDIFormat>(bioXASDefaultXAS->id());
 }
 
-void BioXASAppController::initializeStorageRing()
-{
-	CLSStorageRing::sr1();
-}
-
-void BioXASAppController::initializeBeamline()
-{
-	BioXASBeamline::bioXAS();
-}
-
-void BioXASAppController::initializePeriodicTable()
-{
-	AMPeriodicTable::table();
-}
-
-void BioXASAppController::setupUserInterface()
+void BioXASAppController::setupAcquamanUserInterface()
```

Comment:
@helfrij it is not. When doing code refactor, I want to make thing clear that we have a "generic" function to setup the UI. I can change it back for sure.